packages/medusa/src/api/routes/admin/reservations/index.ts

```diff
@@ -67,6 +69,25 @@ export type AdminReservationsRes = {
   reservation: ReservationItemDTO
 }
 
+/**
+ * @schema ExtendedReservationItem
+ * type: object
+ * allOf:
+ *   - $ref: "#/components/schemas/ReservationItemDTO"
+ *   - type: object
+ *     properties:
+ *       line_item:
+ *         description: optional line item
+ *         $ref: "#/components/schemas/LineItem"
+ *       inventory_item:
+ *         description: inventory item from inventory module
+ *         $ref: "#/components/schemas/InventoryItemDTO"
+ */
+export type ExtendedReservationItem = ReservationItemDTO & {
```

Comment:
same as the previous comment, but probably not as part of that pr but as we move towards modules